src/components/BottomBar/Origin/FocusEntry.jsx

```diff
@@ -95,47 +56,19 @@ function FocusEntry({
       {showNavigationButtons && (
         <div className={styles.buttonContainer}>
           {isActive() && (
-            <Button className={styles.quickAccessFlyTo} onClick={zoomToFocus} title="Zoom to">
-              <MdCenterFocusStrong className={styles.buttonIcon} />
-            </Button>
+            <NavigationButton type="frame" {...navButtonProps} />
           )}
-          <Button className={styles.quickAccessFlyTo} onClick={flyTo} title="Fly to">
-            <MdFlight className={styles.buttonIcon} />
-          </Button>
+          <NavigationButton type="fly" {...navButtonProps} />
           <TooltipMenu
             sourceObject={<MdMoreVert className={styles.buttonIcon} />}
           >
             <SmallLabel className={styles.menuTopLabel}>{identifier}</SmallLabel>
             <HorizontalDelimiter />
-            <Button className={styles.flyToButton} onClick={select} title="Focus">
-              <Row>
-                <SvgIcon className={styles.buttonIcon}><Focus /></SvgIcon>
-                <span className={styles.menuButtonLabel}> Focus </span>
-              </Row>
-            </Button>
+            <NavigationButton type="focus" showLabel {...navButtonProps} />
             <HorizontalDelimiter />
-            <Button className={styles.flyToButton} onClick={flyTo} title="Fly to">
-              <Row>
-                <MdFlight className={styles.buttonIcon} />
-                <span className={styles.menuButtonLabel}> Fly to </span>
-              </Row>
-            </Button>
-            <Button className={styles.flyToButton} onClick={fadeTo} title="Jump to">
-              <Row>
-                <MdFlashOn className={styles.buttonIcon} />
-                <span className={styles.menuButtonLabel}> Jump to </span>
-              </Row>
-            </Button>
-            <Button className={styles.flyToButton} onClick={zoomToFocus} title="Zoom to">
-              <Row>
-                <MdCenterFocusStrong className={styles.buttonIcon} />
-                <span className={styles.menuButtonLabel}> Zoom to / Frame </span>
-                <InfoBox
-                  text={`Focus on the target object by moving the camera in a straigt line
-                  and rotate towards the object`}
-                />
-              </Row>
-            </Button>
+            <NavigationButton type="fly" showLabel {...navButtonProps} />
```

Comment:
Maybe make the type strings into an enum that is exported from the NavigationButton file? This would prevent bugs from typos